Mathlib/Algebra/Order/Ring/Unbundled/Rat.lean

```diff
@@ -28,19 +28,14 @@ namespace Rat
 
 variable {a b c p q : ℚ}
 
-@[simp] lemma mkRat_nonneg {a : ℤ} (ha : 0 ≤ a) (b : ℕ) : 0 ≤ mkRat a b := by
-  simpa using divInt_nonneg ha (Int.natCast_nonneg _)
+@[simp] lemma mkRat_nonneg {a : ℤ} (ha : 0 ≤ a) (b : ℕ) : 0 ≤ mkRat a b :=
+  divInt_nonneg ha <| Int.natCast_nonneg _
```

Comment:
I think this is a bad change, because it relies on the definitional equality between `Rat.divInt` and `mkRat`.